rfc/rfc008-certificate-structure.md

```diff
@@ -70,3 +70,8 @@ A PKIo certificate is costly and it's associated with a single private key. Appr
 
 In case of theft or loss of a private key and/or certificate, please consult the TSP documentation.
 
+## 6. Governance
+
+All connections mentioned in §3 MUST be accepted. A node operator MAY NOT deny a connection request other than those using a banned or revoked certificate. 
```

Comment:
what about throtteling? E.g. a peer flooding the local node with connections?

Reply:
Maybe add a new RFC for a code-of-conduct? Then also place this part in that RFC.